addons/mass_mailing/models/mass_mailing.py

```diff
@@ -360,6 +360,11 @@ class MassMailing(models.Model):
     _inherits = {'utm.source': 'source_id'}
     _rec_name = "source_id"
 
+    @api.model
+    def _get_default_mail_server_id(self):
+        server_id = self.env['ir.config_parameter'].sudo().get_param('mass_mailing.mail_server_id')
```

Comment:
Couldn't we use a literal eval ? + check if someone enters another thing than an int and if the record does not exists anymore.

Reply:
@tde-banana-odoo Changes done.